content/en/blog/2022/how-to-get-started-with-opentelemetry-go.md

```diff
@@ -0,0 +1,421 @@
+---
+title: Go Web App & PpenTelemetry Instrumentation
```

Comment:
There's a typo in the title: PpenTelemetry.

How about the shorter title of:

# Go Web App Instrumentation

Reply:
Oh, I see that you've put that as the `linkTitle`. If you agree with the change mentioned above, then drop the `linkTitle` field.

Reply:
Or maybe the `linkTitle` could be just: **Go App Instrumentation**